qa/integration/services/helpers.sh

```diff
@@ -13,14 +13,29 @@ setup_install_dir() {
 
 wait_for_port() {
     count=$PORT_WAIT_COUNT
-    port=$1
-    while ! nc -z localhost $port && [[ $count -ne 0 ]]; do
+    while ! test_port "$1"  && [[ $count -ne 0 ]]; do
         count=$(( $count - 1 ))
         [[ $count -eq 0 ]] && return 1
         sleep 0.5
     done
     # just in case, one more time
-    nc -z localhost $port
+    test_port "$1"
+}
+
+test_port() {
+    if command -v ncd 2>/dev/null; then
```

Comment:
Should we use ncd here or just 'nc', e.g:

multipass@ls1:~$ command -v nc
/bin/nc

The above is in Ubuntu. So I guess the current works on centos but not ubuntu ?

Reply:
Or maybe use 'which':

multipass@ls1:~$ which nc
/bin/nc
multipass@ls1:~$ which random-name